src/runtime/lpython_builtin.py

```diff
@@ -173,3 +173,13 @@ def oct(n: i32) -> str:
         n = n//8
         res += _values[remainder]
     return prep + res[::-1]
+
+
+def round(value: f64) -> i32:
+    """
+    Rounds a floating point number to the nearest integer.
+    """
+    if abs(value - int(value)) <= 0.5:
+        return int(value)
+    else:
+        return int(value) + 1
```

Comment:
Isn't round the same thing as `int(value+0.5)`? Possibly ±1 for negative values?

Reply:
> Isn't `round` the same thing as `int(value+0.5)`? 

Actually, this gives 1 when the `value` is 0.5, but Python returns 0 for `round(0.5)`

Reply:
Interesting, you are right:
```
>>> round(0.5)
0
```

Reply:
Here is the CPython's implementation: https://github.com/python/cpython/blob/5ab745fc51e159ead28b523414e52f0bcc1ef353/Objects/floatobject.c#L1058, it looks like they do something like this:
```c
        rounded = round(x); // the same as int(x+0.5) ?
        if (fabs(x-rounded) == 0.5)
            rounded = 2.0*round(x/2.0);
```